docs/src/main/asciidoc/context-propagation.adoc

```diff
@@ -135,7 +135,7 @@ them to the client as JSON using xref:rest-json.adoc[Jackson or JSON-B]:
         // Create a REST client to the Star Wars API
         WebClient client = WebClient.create(vertx,
                          new WebClientOptions()
-                          .setDefaultHost("swapi.dev")
+                          .setDefaultHost("swapi.net")
```

Comment:
The title of the PR says `tech` and the quickstart change says `tech` but here we have a `net`? Is it expected?

Reply:
Good catch @gsmet, it should be `.tech` as commit message says, I will create fix PR soon.